class-debug-bar-cron.php

```diff
@@ -380,6 +381,17 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by schedules interval.
+		 *
+		 * @param array $a first element of comparison  pair
+		 * @param array $b second element of comparison  pair
```

Comment:
Capitals at start of comment `first` => `First` + period at the end. (Oh and let's get rid of the extra space `comparison pair`)